client/components/MainDescription.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { mount, shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import MainDescription from './MainDescription.jsx';
+
+configure({ adapter: new Adapter() });
+
+describe('Make sure that the component has an array of descriptions as its state', () => {
+  it('Make sure that the componentDidMount is successfully called', () => {
+    const wrapper = shallow(<MainDescription />);
+    const props = wrapper.props;
+
+    expect(props).toBe(1);
+  });
+});
```

Comment:
hey lmk how this works out going forward.  i ran into some async issues with shallow & mounting my App for multiple tests.  turns out it was because my componentDidMount was making API requests and they weren't finishing by the time my tests would complete.

Reply:
Oops. This test was not working for me either, should not have pushed it up....